client/ayon_maya/plugins/load/load_layout.py

```diff
@@ -0,0 +1,233 @@
+from maya import cmds
+import math
+import json
+import collections
+import ayon_api
+from ayon_maya.api import plugin
+from ayon_maya.api.lib import unique_namespace
+from ayon_core.pipeline import (
+    load_container,
+    get_representation_path,
+    discover_loader_plugins,
+    loaders_from_representation,
+    get_current_project_name
+)
+from maya.api import OpenMaya as om
+from ayon_maya.api.pipeline import containerise
+
+
+class LayoutLoader(plugin.Loader):
+    """Layout Loader(json)"""
+
+    product_types = {"layout"}
+    representations = {"json"}
+
+    label = "Layout Loader(json)"
```

Comment:
Pretty sure the `(json)` is redundant because it will show added to the label in the loader anyway?
```suggestion
    label = "Load Layout"
```